Description: Don't let upstream's CMake fiddle with .symbols file in debian/ folder.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Forwarded: not-needed, Debian-specific

--- a/src/miroil/CMakeLists.txt
+++ b/src/miroil/CMakeLists.txt
@@ -61,20 +61,20 @@
 
 execute_process(COMMAND uname -m OUTPUT_VARIABLE MIR_ARCH)
 
-find_program(MIR_DPKG_GENSYMBOLS dpkg-gensymbols)
-if (CMAKE_COMPILER_IS_GNUCXX   AND  # clang generates slightly different symbols (but we don't care)
-    MIR_USE_LD STREQUAL "ld"   AND  # lld on 22.04 generates slightly different symbols (but we don't care)
-    MIR_ARCH MATCHES 64        AND  # On 32bit platforms int64_t presents as `long long` (but we've `long`)
-    MIR_DPKG_GENSYMBOLS)            # Using dpkg-gensymbols only makes sense on Debian based distros
-
-    if (MIR_DPKG_GENSYMBOLS)
-        add_custom_target(check-miroil-symbols ALL
-            DEPENDS miroil ${PROJECT_SOURCE_DIR}/debian/libmiroil${MIROIL_ABI}.symbols
-            COMMAND python3 ${CMAKE_CURRENT_SOURCE_DIR}/check-and-update-debian-symbols.py ${CMAKE_LIBRARY_OUTPUT_DIRECTORY} miroil ${MIR_VERSION} ${MIROIL_ABI} ${MIROIL_VERSION_MINOR}
-            WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}"
-            VERBATIM)
-    endif()
-endif()
+#find_program(MIR_DPKG_GENSYMBOLS dpkg-gensymbols)
+#if (CMAKE_COMPILER_IS_GNUCXX   AND  # clang generates slightly different symbols (but we don't care)
+#    MIR_USE_LD STREQUAL "ld"   AND  # lld on 22.04 generates slightly different symbols (but we don't care)
+#    MIR_ARCH MATCHES 64        AND  # On 32bit platforms int64_t presents as `long long` (but we've `long`)
+#    MIR_DPKG_GENSYMBOLS)            # Using dpkg-gensymbols only makes sense on Debian based distros
+#
+#    if (MIR_DPKG_GENSYMBOLS)
+#        add_custom_target(check-miroil-symbols ALL
+#            DEPENDS miroil ${PROJECT_SOURCE_DIR}/debian/libmiroil${MIROIL_ABI}.symbols
+#            COMMAND python3 ${CMAKE_CURRENT_SOURCE_DIR}/check-and-update-debian-symbols.py ${CMAKE_LIBRARY_OUTPUT_DIRECTORY} miroil ${MIR_VERSION} ${MIROIL_ABI} ${MIROIL_VERSION_MINOR}
+#            WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}"
+#            VERBATIM)
+#    endif()
+#endif()
 
 configure_file(
     ${CMAKE_CURRENT_SOURCE_DIR}/miroil.pc.in
